pkg/services/librarypanels/database.go

```diff
@@ -115,6 +115,10 @@ func (lps *LibraryPanelService) connectDashboard(c *models.ReqContext, uid strin
 // connectLibraryPanelsForDashboard adds connections for all Library Panels in a Dashboard.
 func (lps *LibraryPanelService) connectLibraryPanelsForDashboard(c *models.ReqContext, uids []string, dashboardID int64) error {
 	err := lps.SQLStore.WithTransactionalDbSession(context.Background(), func(session *sqlstore.DBSession) error {
```

Comment:
Something for the future to consider. Using `context.Background()` is only suggested when you don't have any other context. In this case you could use the request context `c.Context.Req.Context()` I think.

Reply:
Good catch! I'll look into it and change.